### PR TITLE
Update the example result messages for using the Environment Check Sc…

### PR DESCRIPTION
**Pull request by roger-ryao**
…ript

Ref. https://github.com/longhorn/longhorn/issues/5653

### CODE REVIEW

**Review by innobead — APPROVED** [review]

File: content/docs/1.5.2/deploy/install/_index.md [hunk 1]
```diff
@@ -61,10 +61,16 @@ curl -sSfL https://raw.githubusercontent.com/longhorn/longhorn/v{{< current-vers
...
+[INFO]  All nodes have unique hostnames.
 [INFO]  Waiting for longhorn-environment-check pods to become ready (0/3)...
 [INFO]  All longhorn-environment-check pods are ready (3/3).
-[INFO]  Required packages are installed.
+[INFO]  MountPropagation is enabled
+[INFO]  Checking kernel release...
+[INFO]  Checking iscsid...
+[INFO]  Checking multipathd...
+[INFO]  Checking packages...
+[INFO]  Checking nfs client...
```
Comment (jillian-maroket) [inline content/docs/1.5.2/deploy/install/_index.md:73]:
```suggestion
[INFO]  Checking NFS client...
```

File: content/docs/1.3.4/deploy/install/_index.md [hunk 2]
```diff
@@ -61,10 +61,12 @@ curl -sSfL https://raw.githubusercontent.com/longhorn/longhorn/v{{< current-vers
 Example result:
 
 ```shell
-[INFO]  Required dependencies are installed.
+[INFO]  Required dependencies 'kubectl jq mktemp' are installed.
+[INFO]  Hostname uniqueness check is passed.
 [INFO]  Waiting for longhorn-environment-check pods to become ready (0/3)...
 [INFO]  All longhorn-environment-check pods are ready (3/3).
 [INFO]  Required packages are installed.
+[INFO]  MountPropagation is enabled.
```
Comment (jillian-maroket) [inline content/docs/1.3.4/deploy/install/_index.md:69]:
```suggestion
[INFO] Mount propagation is enabled.
```

Comment (jillian-maroket) [reply content/docs/1.3.4/deploy/install/_index.md:69]: I assume that you are referring to the feature and not a specific field or function in the code. If my assumption is correct, we should not use CamelCase here.

**Review by jillian-maroket — COMMENTED** [review]
@roger-ryao Just a few nits. If the changes work for you, please apply them to all affected files. Thanks!

**Pull request merged**